README.md

```diff
@@ -99,10 +99,8 @@ Oh God please yes! :heart: Feel free to look around the [<kbd>help wanted</kbd>]
 3. <span id="f3"></span> Not implemented; tracked in [#17](https://github.com/elm-in-elm/compiler/issues/17)
 4. <span id="f4"></span> Not implemented; not tracked yet
 5. <span id="f5"></span> To be optimized the same way Ints are; not tracked yet
-6. <span id="f6"></span> Comprehensive tests missing; will be fixed in [#15](https://github.com/elm-in-elm/compiler/pull/15)
-7. <span id="f7"></span> Escape sequences not implemented; not tracked yet
-8. <span id="f8"></span> Comprehensive tests missing; not tracked yet
-9. <span id="f9"></span> Multiline strings (and maybe more) missing; not tracked yet
+6. <span id="f6"></span> Comprehensive tests missing; not tracked yet
+7. <span id="f7"></span> Multiline strings (and maybe more) missing; not tracked yet
```

Comment:
Next time please don't bother shifting the numbers, life's too short for that :)

Reply:
Phew :+1: